package.json

```diff
@@ -44,7 +44,7 @@
     "jasmine-node": "~1.14.5",
     "jasmine-reporters": "~1.0.1",
     "jshint-stylish": "~1.0.0",
-    "karma": "~0.12.0",
+    "karma": "^0.12.32-beta.0",
```

Comment:
do we want to use `^`? that will update us automatically to 0.13/0.14 when that is available and somebody does `npm update`. Sounds too risky to me.